lib/commands/sdk.js

```diff
@@ -917,6 +918,22 @@ SdkSubcommands.install = {
 			}
 
 			if (data.file) {
+				// Avoid extracting the whole SDK if we already have an SDK with the same version and timestamp installed
+				if (!cli.argv.force) {
+					// Pull out the version and os from the zipfile name!
+					var m = data.file.match(/mobilesdk\-(\d\.\d\.\d(\.\w+)?)\-(osx|win32|linux)\.zip$/);
```

Comment:
This regex can be improved:

    /^mobilesdk-(\d+\.\d+\.\d+(?:\.\w+)?)-(osx|win32|linux)\.zip$/